server/src/cloud/board.ts

```diff
@@ -144,7 +149,14 @@ export const initializeBoardFunctions = () => {
       };
       return acc;
     }, {});
-    const savedBoard = await board.save({...request, columns, voteLimit: 10, votingIteration: 0, owner: user, showNotesOfOtherUsers: true}, {useMasterKey: true});
+
+    const userConfigurations: UserConfigurations = {};
+    userConfigurations[user.id] = {};
```

Comment:
Shouldnt the default be sth like `{ready: false, showHiddenColumns: false}`?

Reply:
Yes that's correct, but as mentioned above this will be added in the PR individually.